### ISSUE TITLE
aliased provider values in configuration_aliases in required_providers block not supported

### ISSUE THREAD

**Issue by grimm26** [terraform-docs/terraform-docs#473]
### Describe the bug

terraform v0.15.x has introduced a `configuration_aliases` setting in the `required_providers` block.
The aliased provider values in configuration_aliases cause an error.

See https://www.terraform.io/upgrade-guides/0-15.html#alternative-provider-configurations-within-modules

### How can we reproduce it?

```
$ git diff versions.tf
diff --git a/versions.tf b/versions.tf
index 62961b9..877a668 100644
--- a/versions.tf
+++ b/versions.tf
@@ -1,9 +1,10 @@
 terraform {
-  required_version = ">= 0.13"
+  required_version = "~> 0.15.0"
   required_providers {
     aws = {
-      source  = "hashicorp/aws"
-      version = ">= 3.28.0"
+      source                = "hashicorp/aws"
+      version               = ">= 3.28.0"
+      configuration_aliases = [aws.this]
     }
   }
 }
$ terraform-docs markdown .
Error: Variables not allowed: Variables may not be used here.
```

### Environment information

terraform-docs version v0.12.1 darwin/amd64 BuildDate: 2021-03-30T07:26:23+0100
Terraform v0.15.0
on darwin_amd64

**Comment by wagnst** [comment]
Facing the same issue

**Comment by adamjohnson01** [comment]
It looks like this can be fixed by updating go.mod

```
-       github.com/terraform-docs/terraform-config-inspect v0.0.0-20210126151735-6ef25af8884f
+       github.com/terraform-docs/terraform-config-inspect v0.0.0-20210318143659-b932ca5358a6
```

**Comment by bappr** [comment]
And will need to update the github action too. Failing there.

**Comment by adamjohnson01** [comment]
I am happy to create a pr for the changes if nobody else is planning to.

**Comment by bappr** [comment]
@khos2ow when can we expect the repo to be tagged so that we can change the GHA and have it working again?

**Comment by khos2ow** [comment]
This is now available in [v0.13.0](https://github.com/terraform-docs/terraform-docs/releases/tag/v0.13.0).

**Comment by shoekstra** [comment]
Was there a regression?

```shell
$ cat versions.tf
terraform {
  required_version = ">= 0.14"

  required_providers {
    aws = {
      configuration_aliases = [aws, aws.route53]
      source                = "hashicorp/aws"
    }
  }
}

$ terraform-docs markdown .
Error: Variables not allowed: Variables may not be used here. (and 1 other messages)

$ terraform-docs -v
terraform-docs version v0.16.0 darwin/amd64
```
